File: problems/1207 - Unique Number of Occurrences/PHP/Solution.php
<?php

class Solution {

    /**
     * @param Integer[] $arr
     * @return Boolean
     */
    function uniqueOccurrences($arr) {
        $m = [];
        $m1 = [];
        foreach ($arr as $item) {
            $m[$item]++;
            
        }

        foreach ($m as $v) {
            $m1[$v] = true;
        }

        return count($m) === count($m1);
    }
}

File: problems/1207 - Unique Number of Occurrences/Go/Solution.go
func uniqueOccurrences(arr []int) bool {
	m := map[int]int{}
	m1 := map[int]bool{}

	for _, v := range arr {
		m[v] = m[v] + 1
	}

	for _, value := range m {
		m1[value] = true
	}

	return len(m) == len(m1)
}